en/computer_vision/obstacle_avoidance.md

```diff
@@ -56,11 +54,41 @@ The difference when avoidance is active are:
 If PX4 stops receiving setpoint updates then obstacle avoidance will be disabled, and the mission continues under normal PX4 [Mission mode](../flight_modes/mission.md) control. 
 
 
-###  Mission Mode Avoidance Interface {#mission_avoidance_interface}
+### Path Planning Interface {#path_planning_interface}
```

Comment:
FYI, this is a restructure, but not quite finished. Basically what the companion sends to PX4is always the same, so I put that first. However i THINK that PX4 probably sends something different in mission mode or land mode outside of missions, because the desired path is different in each case. So that is now second, and I'll expand that out IFF the assumption is correct (and if not will revert)